articles/media-services/latest/stream-files-dotnet-quickstart.md

```diff
@@ -65,8 +65,9 @@ For explanations about what each function in the sample does, examine the code a
 
 When you run the app, URLs that can be used to playback the video using different protocols are displayed. 
 
-1. Press Ctrl+F5 to run the *EncodeAndStreamFiles* application.
-2. Choose the Apple's **HLS** protocol (ends with *manifest(format=m3u8-aapl)*) and copy the streaming URL from the console.
+1. Open AMSV3Quickstarts in VSCode.
+2. Press Ctrl+F5 to run the *EncodeAndStreamFiles* application with .NET. This may take a few minutes.
```

Comment:
One quick question - I cant recall, does the customer have to go to the launch settings and choose that project or does that one run by default when hitting CTL+F5